dotnet/hello_world/Program.cs

```diff
@@ -0,0 +1,210 @@
+﻿using Google.Cloud.Bigtable.Admin.V2;
+using Google.Cloud.Bigtable.V2;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Google.Protobuf;
+using System.Linq;
+using System.IO;
+using System;
+
+
+namespace HelloWorld
+{
+    class Program
+    {
+        // Connect to bigtable and perform basic operations
+        // 1. Create table with column family
+        // 2. Insert rows
+        // 3. Read rows
+        // 4. Delete table
+        //String[] x = Environment.GetCommandLineArgs();
+
+        static HelloWorldSettings _settings;
+
+        static void Main(string[] args)
+        {
+            #region Setting up application settigns
+            var builder = new ConfigurationBuilder().
+                SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
+
+            IConfigurationRoot configuration = builder.Build();
+
+            _settings = new HelloWorldSettings();
+            configuration.Bind(_settings);
+            #endregion
+
+            HelloWorld helloWorld = new HelloWorld(_settings.ProjectId, _settings.InstanceId);
+            helloWorld.DoHelloWorld();
+            Console.WriteLine("Press any key to exit");
+            Console.ReadKey();
+        }
+
+        public class HelloWorld
+        {
+
+            public string ProjectId { get; set; }
+            public string InstanceId { get; set; }
+
+            public HelloWorld(string projectId, string instanceId)
+            {
+                ProjectId = projectId;
+                InstanceId = instanceId;
+            }
+
+            public void DoHelloWorld()
+            {
+
+                // Build Instance Name
+                InstanceName instanceName = new InstanceName(ProjectId, InstanceId);
```

Comment:
I'm curious why this is `InstanceName` — I would think an instance name is a string, but this is more than that. Should this be `Instance` instead?

Reply:
I only chose `InstanceName` over `Instance` because this is type of the parameter needed for [CreateTable()][CreateTable()] and [ListTable()][ListTable()] methods

[CreateTable()]:https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Bigtable.Admin.V2/Google.Cloud.Bigtable.Admin.V2/BigtableTableAdminClient.cs#L744
[ListTable()]:https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Bigtable.Admin.V2/Google.Cloud.Bigtable.Admin.V2/BigtableTableAdminClient.cs#L1061